shanoir-ng-datasets/src/main/java/org/shanoir/ng/datasetacquisition/controler/DatasetAcquisitionApi.java

```diff
@@ -50,6 +51,20 @@
 @Api(value = "datasetacquisition", description = "the datasetacquisition API")
 public interface DatasetAcquisitionApi {
 	
+	@ApiOperation(value = "", notes = "Creates new dataset acquisition", response = Void.class, tags={  })
+    @ApiResponses(value = {
+        @ApiResponse(code = 204, message = "created Dataset Acquitistion", response = Void.class),
+        @ApiResponse(code = 401, message = "unauthorized", response = Void.class),
+        @ApiResponse(code = 403, message = "forbidden", response = Void.class),
+        @ApiResponse(code = 422, message = "bad parameters", response = ErrorModel.class),
+        @ApiResponse(code = 500, message = "unexpected error", response = ErrorModel.class) })
+    @PostMapping(value = "/datasetacquisition/new",
+        produces = { "application/json" },
+        consumes = { "application/json" })
+    ResponseEntity<DatasetAcquisition> createNewDatasetAcquisition(
```

Comment:
same here please: saveNewDatasetAcquisition

Reply:
for security reasons: please add an PreAuthorize here for role admin, as nobody calls this normally from inside one instance